content/post/web-analytics-process.md

```diff
@@ -0,0 +1,83 @@
+
+I suggest using these alerts to look only for data outages, rather than to surface traffic changes that could be within the normal range of behavior. This is to avoid the fatigue of too many emails, causing you to not notice, or completely disregard, alerts. If you still like to use alerts for other use cases, you can consider sending them to another email address.
+
+### Step 6: Create documentation and training
+Document as much as you can: the current setup, what is being tracked, any technical notes, who is responsible for implementation or questions, dates of any outages or site changes. The documentation should contain details of account customizations like custom dimensions, event tracking, and marketing campaign tags. This information should be kept in a shared document online or on your intranet.
+
+Training should be handled by whoever is responsible for web analytics at your organization, whether that is someone onsite or at a partnered agency. 
+
+Regular training is important to empower employees and build analytics into the culture of the organization. You may discover, as I have, pent-up demand for web analytics training, where training sessions fill up well in advance from employees eager to learn how to understand and improve their own performance through data. 
+
+### Step 7: Run mini-audits
+Running a full audit regularly is not practical due to the effort involved, but it is still important to keep on top of changes. To address this you can run regular mini audits every month or two to review and catch tracking and setup problems that might have otherwise been overlooked. Google lists [several tools](https://www.google.com/analytics/partners/search/apps?category=114) to automate the process (I created one of them, the [Online Audit Tool for Google Analytics](https://www.google.com/analytics/partners/company/5118885300797440/gadp/5629499534213120/app/5707702298738688/listing/5639274879778816), but there are several to choose from).
+
+## Final thoughts 
+Ideally everyone should be looking at their slice of the business, not just designated Data Analysts. Besides the business benefits of democratizing and increasing data usage, having more eyes on the data will help quickly identify any broken tracking or other issues. And if there are no tracking issues, then people will put more faith in the data and use it more, which is the desired cycle. Following the above steps will help prevent the scenario where the web analytics tool is distrusted and avoided, and instead land your organization at a place where employees view the tool positively, as a useful and trusted resource that can support decision making.
```

Comment:
For this sentence: 
> And if there are no tracking issues, then people will put more faith in the data and use it more, which is the desired cycle.
It'd be great to get a back-link to my post on the data quality flywheel :)